### PR TITLE
Add new blog on data quality facets and Great Expectations

### PR DESCRIPTION
**Pull request by collado-mike**
Signed-off-by: Michael Collado <mike@datakin.com>

### CODE REVIEW

File: contents/blog/dataquality_expectations_facet/index.mdx [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+the incredible feature was unleashed. For days afterward, everyone watched the metrics dashboards waiting for that
+hockey stick uptick in the revenue graphs 📈. But it never materialized! The graph was flat. No change at all! Weeks
+went by. Nothing. How could the experiment results be so far from reality?
+
+Of course, if you’ve ever run A/B tests, you probably already recognized the developer’s mistake. In their randomization
+logic, customers who were assigned control were logged and forgotten, while customers who were assigned treatment were
+logged only after validating that they owned a Kindle device. It turned out the total number of customers who came to
+amazon.com was far greater than the number of customers who owned a Kindle device. And if you divide the total sum of
+Kindle book sales by all of the amazon.com customers, regardless of whether they own a Kindle device, that average
+will come out quite a lot lower than if you calculate the average Kindle book revenue from only customers who own Kindles.
```
Comment (julienledem) [inline contents/blog/dataquality_expectations_facet/index.mdx:41]: 🤣

File: contents/blog/dataquality_expectations_facet/index.mdx [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+
+## Data-Driven Depends On Data-Quality
+
+Today, most businesses consider themselves data-driven. The stereotype of the maverick CEO leading with his or her gut
+is mostly antiquated, with a handful of exceptions. And yet, even though people know intellectually that data is only
+useful if it is correct, we tend to stop digging once we find some data that confirms our pre-existing assumptions. We
+justify bad decisions by claiming that they are “data-based” without ever validating the quality of our sources. Where
+did that data come from? How old is it? Is the logic that generated it correct? Is it derived from some other dataset?
+What is the quality of that dataset?
+![Data-driven: You keep using that word. I do not think it means what you think it means](./data_driven.png)
```
Comment (julienledem) [inline contents/blog/dataquality_expectations_facet/index.mdx:60]: I like it :)
also: https://twitter.com/J_/status/1403052224293396482?s=20

File: contents/blog/dataquality_expectations_facet/index.mdx [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+    {
+      "expectation_type": "expect_column_values_to_be_unique",
+      "kwargs": {
+        "column": "email"
+      },
+      "meta": {}
+    }
+  ],
+  "meta": {
+    // ...
```
Comment (julienledem) [inline contents/blog/dataquality_expectations_facet/index.mdx:127]: Does it also show up in red once rendered by gatsby? maybe not.

File: contents/blog/dataquality_expectations_facet/index.mdx [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+model with useful information about the jobs, runs, and datasets reported on. One interesting detail about facets is
+that they can be attached to an entity after the fact. In the Marquez reference implementation, a dataset version is
+created every time a job run writes to or otherwise modifies a dataset. _Output_ facets, such as the new record count or
+the number of bytes written, are attached directly to the dataset version when the job run completes. But consuming
+jobs can also attach facets to the version of the dataset that exists at the start time of the job’s execution.
+
+In the OpenLineage Airflow integration, Great Expectations tasks, such as the one in our example DAG above, are
+evaluated after they run and the expectation results (as well as some other data quality metrics) are collected
+into a
+[DataQualityMetricsInputDatasetFacet](https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.json#L446-L499)
```
Comment (julienledem) [inline contents/blog/dataquality_expectations_facet/index.mdx:236]: maybe the link text can be the readable form: 
```
DataQualityMetrics input dataset facet
```

**Review by julienledem — APPROVED** [review]
I like the memes! :)

**Review by rossturk — APPROVED** [review]

**Comment by collado-mike** [comment]
> I like the memes! :)

It occurred to me the memes peg my age pretty much exactly 😂

File: contents/blog/dataquality_expectations_facet/index.mdx (hunk 3 above)
Comment (collado-mike) [reply contents/blog/dataquality_expectations_facet/index.mdx:127]: Looks like this in Gatsby
Screen Shot 2021-08-10 at 10 26 48 AM

Comment (julienledem) [reply contents/blog/dataquality_expectations_facet/index.mdx:127]: looks good

**Pull request merged**